src/content/docs/mobile-monitoring/new-relic-mobile-ios/watchOS/watchOS-unsupported-features.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: WatchOS unsupported features
+type: troubleshooting
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - Troubleshoot
+  - watchOS
+metaDescription: 'The features currently unsupported by the watchOS agent'
+freshnessValidatedDate: never
+---
+
+Apple does not provide the all the same APIs to watchOS platform. Because of this some of the features that the iOS agent provides are not available on watchOS. We seek to have parity in our platforms so as Apple releases new functionality we will continue to investigate solutions.
```

Comment:
This doc could be another section inside the get started with new relic watchOS doc above. What do you think about adding that as another h2?

Reply:
That works for me.